src/config.rs

```diff
@@ -72,8 +72,8 @@ pub struct GitDependency {
     pub rev: Option<String>,
 }
 
-impl core::fmt::Display for GitDependency {
```

Comment:
Any reason for changing `core::fmt` to `std::fmt`? The latter is a re-export

Reply:
I prefer using `std` when possible, and `core`/`alloc` when `no_std` is enabled